### DISCUSSION TITLE
what happened to the Xab plugin?

### DISCUSSION THREAD

**Discussion by Max-Lovell** [jspsych/jsPsych#1939] [Q&A] [answered]
this plugin here: https://github.com/jodeleeuw/jsPsych-GUI/blob/master/jspsych-5.0.1/jspsych-5.0.1/docs/markdown_docs/plugins/jspsych-xab.md
is used in some code I am rewriting, written by someone else. Does anyone know if there is a plugin with the same functionality in the current version of jsPsych as I can't seem to find it?

Thanks!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @Max-Lovell,

I removed it from more recent versions of jsPsych because I thought that in most cases pairing two `html-keyboard-response` (or `image-keyboard-response`) trials in sequence can accomplish the same thing, but with more control. Given the redundancy, I didn't want to maintain/update the XAB plugin.

**Reply by Max-Lovell** [reply]
great, thanks for the quick reply!
